crates/oxc_linter/src/rules/unicorn/prefer_query_selector.rs

```diff
@@ -127,9 +128,19 @@ impl Rule for PreferQuerySelector {
                     // argument_expr.span().source_text(ctx.source_text());
                     let source_text = fixer.source_range(argument_expr.span());
                     let quotes_symbol = source_text.chars().next().unwrap();
-                    let sharp = if cur_property_name.eq(&"getElementById") { "#" } else { "" };
+                    let argument = match *cur_property_name {
+                        "getElementById" => format!("#{literal_value}"),
+                        "getElementsByClassName" => format!(
+                            ".{}",
+                            Regex::new(r"\s+").unwrap().replace_all(literal_value, " .")
```

Comment:
We should avoid `Regex` at all cost, a quick split and then filter out whitespace should also work.

Reply:
I’ll change this when I get a chance, if somebody else gets to it sooner feel free to close this out.
Marking as a draft for now.